todoapp/todos/utils.py

```
@@ -51,11 +66,25 @@ def fetch_all_users():
 # use json.load(json.dumps(serializer.data)) while returning data from this function for test cases to pass.
 def fetch_all_todo_list_with_user_details():
```

Ruff (0.12.2)

Missing return type annotation for public function `fetch_all_todo_list_with_user_details`

(ANN201)